### PR TITLE
DOC-6983 GH workflow docs

### PR DESCRIPTION
**Pull request by urbiz-nr**
Moved GH FAQs doc to the style guide:
- Created new doc Get around with basics on the board/roles/columns
- Updated writer workflow
- Created peer editor workflow doc
- Created troubleshooting doc

Peer editor: please het in touch if you have any doubts/questions!

### CODE REVIEW

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Hero: to do**
+      </td>
+      <td>
+        PRs that the Hero needs to review, publish, and  follow up with SMEs as needed.  
+        
+        **Assign yourself as Assignee**.
```
Comment (x8a) [inline src/content/docs/style-guide/writer-workflow/github-intro.mdx:57]: Maybe it’s worth specifying that it’s the hero who will turn into the Assignee: `Hero: assign yourself as Assignee.`

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+</table>
+
+As a **Hero or Sidekick**, make sure you attend to the following throughout your day:
+* Check in with the Hero/Sidekick at the start of your day (especially on Monday at the start of the week). Don’t forget to sync with the BCN hero if necessary.
+* Watch for incoming PRs in #docs_deploys, and review everything in the Needs triage column. Drag cards from that column to the appropriate column.
+* Work through the cards in the  Hero: to do column.
+
+**Everyone on the team** helps keep things moving:
+* All writers should keep an eye on the Any TW columns. 
+* There's one column for PRs that need a simpler review before merging (typo fixes, drive-by edits, etc), and another column for PRs that need a peer edit.
```
Comment (x8a) [inline src/content/docs/style-guide/writer-workflow/github-intro.mdx:128]: Maybe we could mention the name of the column? `Any TW: Writer needs PR review`

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+* There are also two blocked columns: One for PRs blocked on a SME, and another column where we're waiting on the TW who created the PR to review feedback and/or merge.
```
Comment (x8a) [inline src/content/docs/style-guide/writer-workflow/github-intro.mdx:129]: Would it make sense to move this in the board columns table `Waiting on SME`?

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      </td>
+      <td>
+        Like our Needs Peer Edit column in Jira: A writer has requested a review of their PR. Review their PR in GitHub and leave comments. 
+        
+        **Whoever takes it: assign yourself as Reviewer.**
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Waiting on SME**
```
Comment (x8a) [inline src/content/docs/style-guide/writer-workflow/github-intro.mdx:92]: Should rename it to `Waiting on SME/Blocked`?

File: src/content/docs/style-guide/writer-workflow/tech-writer-workflow.mdx [hunk 5]
```diff
@@ -7,66 +7,102 @@ topics:
...
   - `npm install -g yarn`
 
-### Build the site
-1. In your Terminal, go to your cloned repo, **docs-website**. 
+### Build the site [#build]
+1. In your terminal, go to your cloned repo, **docs-website**. 
     - `cd ~/Documents/github/docs-website`
 2. Run yarn with the following commands:
     - `yarn`
     - `yarn start`
```
Comment (x8a) [inline src/content/docs/style-guide/writer-workflow/tech-writer-workflow.mdx:85]: Can’t we say to just run ‘yarn && yarn start’?

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+### Copy edit workflow [#edit-copy]
+
+If you have copy edits for a file rather than comments and suggestions, you can make the changes to the file in different ways. Here are two main options:
+
+1. Edit using the GitHub browser: On the **Files changed** tab, in the diff window click the editing button (three dots). When you finish your edits, add a comment at the bottom of the file and choose to either **commit the changes directly**, or **create a branch and start a pull request**. Choose to branch and start a pull request if you expect a writer to review the diff and accept or revise your edits.
+2. Edit locally: Check out the branch containing the file you want to edit. In GitHub Desktop, click the Current branch down arrow and select the branch. Then, make the edits on your local drive, save, and commit your changes to the branch. 
+    - Note that this approach adds your edits to the open pull request. You can now see the changes you added to the file on the **Files changed** tab in the PR.
+
+These are just a few of many editing options. You’ll find your preferred way, just as you have with the Drupal tools.
```
Comment (x8a) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:35]: Is it worth it to keep the reference to Drupal? Maybe we can delete this part of the sentence.

**Review by x8a — COMMENTED** [review]
I have some comments, but I love how you organized the information!

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx (hunk 1 above)
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/github-intro.mdx:57]: Good point!

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx (hunk 2 above)
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/github-intro.mdx:128]: It's both columns. I'll clarify that.

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx (hunk 3 above)
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/github-intro.mdx:129]: I think this is more a checklist for writers, I'd keep it here.

File: src/content/docs/style-guide/writer-workflow/tech-writer-workflow.mdx (hunk 5 above)
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/tech-writer-workflow.mdx:85]: Good catch!

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx (hunk 4 above)
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/github-intro.mdx:92]: Yes!

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 6 above)
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:35]: I'll rephrase this to "other tools".

**Review by x8a — APPROVED** [review]

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        **Needs triage**
+      </td>
+      <td>
+        The Hero or Sidekick review issues and PRs in this column, then drag them to the appropriate column.
+
+        If the issue/PR is labeled `eng`, go ahead and click its ellipses icon to archive it. For other issues and PRs, add these labels as applicable:
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/github-intro.mdx:39]: Done!

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+        * `content`
+        * `from_internal`, when issued by a New Relic employee. If you open an PR based on an external issue, label that PR as internal since you're doing the work.
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/github-intro.mdx:41]: This has already been updated.

File: src/content/docs/style-guide/writer-workflow/github-intro.mdx [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+        * `from_external`, when issued by someone outside New Relic, or you're not sure they are an employee.
+        * `Jira’d`, if you create a Jira from the issue. When you create a Jira, don't close the original issue/PR. However, feel free to remove it from the board until you work on it again.
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/github-intro.mdx:43]: Added!

File: src/content/docs/style-guide/writer-workflow/github-troubleshooting.mdx [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+  </Collapser>
+  
+  <Collapser
+    className="freq-link"
+    id="remove-cache"
+    title="Remove corrupted cache files"
+  >
+  There may be times when your .cache directory has been corrupted. This directory is ignored by Git, which means that it travels with you from branch to branch. This might be the problem if your local builds are failing regardless of which branch you’re on.
+  
+  To solve this, run `rm -rf .cache`.
```
[the remark this answers is not in the document]
Comment (urbiz-nr) [reply src/content/docs/style-guide/writer-workflow/github-troubleshooting.mdx:74]: It doesn't hurt to add it!
